[WIP] support get flops and parameters in dev-1.x (#1414)
open-mmlab/mmocr: docs/en/user_guides/useful_tools.md

| config        | str   | (required) Path to the config.    |
| pkl_results   | str   | (required) The saved predictions. |
| --cfg-options | float | Override configs. [Example](<>)   |

> ### Calculate FLOPs and Parameters

Reviewer: ```suggestion
### Calculate FLOPs and the Number of Parameters
```

> We provide a method to calculate the FLOPs and Parameters, first we install the dependencies using the following command.

Reviewer: ```suggestion
We provide a method to calculate the FLOPs and the number of parameters, first we install the dependencies using the following command.
```

```shell
pip install fvcore
```

> The script to calculate the FLOPs and Parameters is used as follows.

Reviewer: ```suggestion
The usage of the script to calculate FLOPs and the number of parameters is as follows.
```

```shell
python tools/analysis_tools/get_flops.py ${config} --shape ${IMAMGE_SHAPE}
```

| ARGS    | Type | Description                                                                                  |
| ------- | ---- | -------------------------------------------------------------------------------------------- |
| config  | str  | (required) Path to the config.                                                               |
| --shape | int  | Image size to use when calculating FLOPs, example usage `--shape 320 320` Default is 640 640 |

> The sample command to get `dbnet_resnet18_fpnc_100k_synthtext.py` FLOPs and the number of parameters is as follows.

Reviewer: ```suggestion
For example, you can run the following command to get FLOPs and the number of parameters of `dbnet_resnet18_fpnc_100k_synthtext.py`:
```
